app/controllers/rents_controller.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class RentsController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    render json: Rent.all.page(params[:page])
+  end
+
+  def create
+    rent = Rent.new(rent_params)
+    rent.save
```

Comment:
If for some reason (like missing params) rent fails to save here, it will fail silently and get rendered as a success, you should render an error if it couldn't be saved